src/main/resources/assets/soot/lang/zh_CN.lang

```diff
@@ -0,0 +1,27 @@
+embers.jei.recipe.dawnstone_anvil=黎明石砧合成
+embers.jei.recipe.alchemical_mixer=炼金台合成
+
+tile.ember_burst.name=Ember Spread Burst
+tile.ember_funnel.name=灰烬漏斗
+tile.alchemy_globe.name=炼金重组器
+
+item.signet_antimony.name=锑纹章
+
+fluid.sugar=熔融糖
+fluid.antimony=熔融锑
+fluid.boiling_wort=熔融麦芽汁
+fluid.boiling_potato_juice=熔融马铃薯汁
+fluid.boiling_wormwood=熔融Verdigris
+fluid.boiling_beetroot_soup=熔融甜菜汁
+fluid.ale=啤酒
+fluid.root_beer=根啤
+fluid.vodka=马铃薯伏特加溶液
+fluid.absinthe=苦艾酒
+fluid.methanol=甲醇
+fluid.umber_ale=茶啤
+
+potion.potency.2=III
+potion.potency.3=IV
+potion.potency.4=V
```

Comment:
Not sure how Minecraft handles the different potion levels in chinese, is it in roman numerals? 
That said, this part isn't too important because the potion effects in soot are still work in progress.

Reply:
Yes. In the Chinese version, potions are like that.
You can find in the Chinese wiki address：https://minecraft-zh.gamepedia.com/%E8%8D%AF%E6%B0%B4

Reply:
ok good to know